app/serializers/project_serializer.rb

```diff
@@ -21,10 +21,15 @@ class ProjectSerializer
     subjects_export: { include: false },
     aggregations_export: { include: false }
   can_filter_by :display_name, :slug, :beta_requested, :beta_approved,
-    :launch_requested, :launch_approved, :private
+    :launch_requested, :launch_approved, :private, :state
   can_sort_by :launch_date, :activity, :completeness, :classifiers_count,
     :updated_at, :display_name
 
+  def self.page(params = {}, scope = nil, context = {})
+    params["state"] = Project.states[params["state"]] if params.key?("state")
```

Comment:
This is the piece that @marten helped me with that solved the enum-related int/string issue. Enums are cool but feel kind of squirrely in practice, especially underneath Restpack.